<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditHouse</class>
 <widget class="QWidget" name="AddEditHouse">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>House Code 2:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="houseCode2LineEdit"/>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="hostLineEdit"/>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="houseCode1LineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>House Code 1:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>hostLineEdit</tabstop>
  <tabstop>houseCode1LineEdit</tabstop>
  <tabstop>houseCode2LineEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>nameLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>okButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hostLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>okButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>houseCode1LineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>okButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>houseCode2LineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>okButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
